apps/builder/app/routes/rest.project.$projectId.tsx

```diff
@@ -1,13 +1,15 @@
 import { json, type LoaderArgs } from "@remix-run/node";
 import type { Data } from "@webstudio-is/react-sdk";
+import { db as projectDb } from "@webstudio-is/project/index.server";
 import { sentryException } from "~/shared/sentry";
 import { loadProductionCanvasData } from "~/shared/db";
 import { createContext } from "~/shared/context.server";
+import { getUserById, type User } from "~/shared/db/user.server";
 
 export const loader = async ({
   params,
   request,
-}: LoaderArgs): Promise<Data> => {
+}: LoaderArgs): Promise<Data & { user?: { email: User["email"] } }> => {
```

Comment:
If we allow more strict typings then your type having how you use it below is not optional but 
`user: { email: User["email"] } | undefined`